extensions/smallrye-jwt/deployment/src/test/resources/applicationJwtCookie.properties

```diff
@@ -0,0 +1,7 @@
+mp.jwt.verify.publickey.location=/publicKey.pem
+mp.jwt.verify.issuer=https://server.example.com
+smallrye.jwt.claims.groups=User
+smallrye.jwt.token.header=Cookie
+smallrye.jwt.token.cookie=a
+quarkus.smallrye-jwt.auth-mechanism=MP-JWT
+quarkus.smallrye-jwt.enabled=true
```

Comment:
There is some strange trailing character here after the true, or is that an indication that there is no newline at the end of the file as I don't see it when going into the raw file view?